shared/actions/chat.js

```diff
@@ -401,13 +407,61 @@ function * _openFolder (): SagaGenerator<any, any> {
 
 function * _newChat (action: NewChat): SagaGenerator<any, any> {
   yield put(searchReset())
-  yield put(searchAddUsersToGroup(action.payload.existingParticipants.map(usernameToSearchResult)))
+
+  const metaDataSelector = (state: TypedState) => state.chat.get('metaData')
+  const metaData = ((yield select(metaDataSelector)): any)
+
+  const following = (yield select(followingSelector)) || {}
+
+  yield put(searchAddUsersToGroup(action.payload.existingParticipants.map(username => ({
+    service: 'keybase',
+    username,
+    isFollowing: !!following[username],
+    extraInfo: {
+      service: 'none',
+      fullName: metaData.getIn([username, 'fullname'], 'Unknown'),
+    },
+  }))))
   yield put(setSearchActive(true))
 }
 
+function * _updateMetadata (action: UpdateMetadata): SagaGenerator<any, any> {
+  const results: any = yield call(apiserverGetRpcPromise, {
+    param: {
+      endpoint: 'user/lookup',
```

Comment:
when you select a conversation we get metadata on the participants so we can fill in the panel